src/src/testruner/hillel/Annotation.java

```diff
@@ -0,0 +1,9 @@
+package testruner.hillel;
+
+import java.lang.annotation.*;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface Annotation {
+    int priority() default 5;
```

Comment:
I would suggest setting the default priority to 0, which would be the highest priority.